Sources/TCP/Descriptor.swift

```diff
@@ -0,0 +1,11 @@
+public struct Descriptor: Hashable {
```

Comment:
If a Descriptor is the equivalent of an Int32, and it's only 2 helpers are on Int32, too. Why do we need a separate struct?